README.md

```diff
@@ -1 +1,89 @@
-# ovirt-ansible-engine-setup
+oVirt Engine Setup
+==================
+
+Install required packages for oVirt Engine deployment, generates answerfile
+and runs engine-setup with it.
+Optionally the role can update oVirt engine packages.
+
+Target Systems
+--------------
+
+* engine
+
+Requirements
+------------
+
+ * Environment with configured repositories.
+ * Ansible version 2.3
+
+Role Variables
+--------------
+
+By default engine-setup uses answer file specific for version of oVirt,
+based on ``ovirt_engine_setup_version`` parameter. You can specify own answer file
+as ``ovirt_engine_setup_answer_file_path``.
+
+| Name                            | Default value         |  Description                                              |
+|---------------------------------|-----------------------|-----------------------------------------------------------|
+| ovirt_engine_setup_version            | UNDEF                 | Allowed versions: [3.6, 4.0, 4.1, 4.2] |
+| ovirt_engine_setup_product_type       | ovirt-engine          | Type of product 'ovirt-engine' or 'rhvm'  | 
+| ovirt_engine_setup_package_list       | [ovirt-engine-dwh-setup,ovirt-engine-extension-aaa-ldap] |  List of extra packages to be installed on engine apart from ovirt-engine package. |
```

Comment:
Why do we need `ovirt-engine-dwh-setup` and `ovirt-engine-extension-aaa-ldap` to be installed by default? I'd say that default should be provided by existing `ovirt-*-setup-*` RPM dependencies, so this list should be empty

Reply:
Didn't read the code nor specs/etc., so not sure if this code should handle updates. If it does, then our docs usually say to:

 yum update ovirt\*setup\*

Which does include ovirt-engine-dwh-setup.

ovirt-engine-setup already requires -dwh-setup, but not a very specific version. So should be updated manually. If user does not, engine-setup will fail, saying 'An update for the Setup packages {packages} was found'.

For clean installs it's enough to install ovirt-engine. That's true also for downstream, BTW, see https://bugzilla.redhat.com/show_bug.cgi?id=1251129